app/src/main/java/com/infomaniak/mail/ui/main/search/SearchFragment.kt

```diff
@@ -131,8 +132,10 @@ class SearchFragment : TwoPaneFragment() {
 
     private fun handleEdgeToEdge(): Unit = with(binding) {
         applyWindowInsetsListener(shouldConsume = false) { _, insets ->
-            appBar.applyStatusBarInsets(insets)
+            toolbar.applySideAndBottomSystemInsets(insets, withBottom = false)
+            toolbar.setMargins(top = insets.safeArea().top) // Apply margins to keep the icon and the search bar aligned
```

Comment:
You can use a `toolbar.apply` to group these two lines even more.

Also, group `chipsList` next to `toolbar` as they are both views contained inside of the app bar layout unlike `swipeRefreshLayout`